homeassistant/components/skybell/__init__.py

```diff
@@ -46,11 +47,21 @@
 
 async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
     """Set up the SkyBell component."""
-    hass.data.setdefault(DOMAIN, {})
-
-    entry_config = {}
     if DOMAIN not in config:
         return True
+
+    hass.data.setdefault(DOMAIN, {})
```

Comment:
The setup entry actually expects this one to always happen. The PR in its current state, the integration will break in case the user has no YAML.